inc/namespace.php

```diff
@@ -45,7 +45,7 @@ function on_plugins_loaded() {
 		Stream\bootstrap();
 	}
 
-	if ( ! empty( $config['2-factor-authentication'] ) ) {
+	if ( ! empty( $config['2-factor-authentication'] ) && Altis\get_environment_type() !== 'local' ) {
```

Comment:
We might want to test out 2fa functionality locally so I wouldn't do this check here, maybe in the override functions themselves.

Reply:
Added a filter and updated docs 👍